### DISCUSSION TITLE
Trigger Syntax for Config Files

### DISCUSSION THREAD

**Discussion by G1tNew** [Azure-Player/azure.datafactory.tools#136] [Q&A] [answered]
Hi,
How do we translate the ARM Template notation for a trigger that is tied to a pipeline into the path in the config file?
For example, in the ARM Template, the trigger looks something like this:

[trigger name]_properties_[pipeline name]_parameters_[pipeline parameter name]

What I am struggling with is that the ARM template syntax shown above does not show the full JSON path for the pipeline reference. If I use the full pipeline reference, like [...].properties.pipeline.pipelineReference.referenceName, the deployment would fail, because the code base doesn't have that same path information.

Any insight is greatly appreciated.
Thank you.
A. Crawford

**Comment by NowinskiK** [comment, marked as answer]
Hi @G1tNew.
This tool supports the deployment (directly) from code (JSON files), NOT from ARM Template files.
So firstly, you should set up repo for the ADF, then use the repo as a source. All files over there are in JSON format.
HTH.
